SignalMessaging/utils/OWSContactAvatarBuilder.m

```diff
@@ -97,8 +90,8 @@ - (id)cacheKey
 
 - (nullable UIImage *)buildDefaultImage
 {
-    UIImage *cachedAvatar =
-        [self.contactsManager.avatarCache imageForKey:self.cacheKey diameter:(CGFloat)self.diameter];
+    UIImage *_Nullable cachedAvatar =
+        [OWSContactAvatarBuilder.contactsManager.avatarCache imageForKey:self.cacheKey diameter:(CGFloat)self.diameter];
```

Comment:
Why did `contactsManager` become a class method? It seems slightly more awkward - no biggie though.

Reply:
Because we want to access it in the initializers, and I don't like to use `self` before base class' initializer is invoked.